src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -490,7 +498,7 @@ public void findOrganisationPbaWithoutEmailByExternalUserShouldBeBadRequest() {
 
     @Test
     @ToggleEnable(mapKey = "OrganisationExternalController.deletePaymentAccountsOfOrganisation", withFeature = false)
```

Comment:
do all the tests with `@ToggleEnable` having `withFeature = false` pass without adding the `@ExtendWith(FeatureToggleConditionExtension.class)` annotation?

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1122/commits/711ccd95c9dc836ae3880e50cc47aa46ddf8ab1b